humility-hiffy/src/lib.rs

```diff
@@ -184,8 +184,14 @@ impl<'a> HiffyContext<'a> {
     }
 
     fn definition(hubris: &'a HubrisArchive, name: &str) -> Result<HubrisGoff> {
+        // Depending on compiler version, definitions may be in either
+        // `self.definitions` or `self.variables`, so we'll check in both
+        // places.  Behavior depends on whether the compiler included DWARF /
+        // ELF debug info for the type of an otherwise-unused variable marked as
```

Comment:
IIUC it's not "whether the compiler included DWARF debug info", the thing is in the DWARF in either case. the difference is whether the _linker_ included a symbol for it (with `#[used(linker)]`) or if it was optimized out at link-time but included in the DWARF (with `#[used(compiler)]`) .